### ISSUE TITLE
Focus state only 1.81:1 contrast ratio - Focus appearance / Non-text contrast

### ISSUE THREAD

**Issue by dashouse** [ThePacielloGroup/inclusive-design-principles#89]
Hello, first off love this site!

The colour #FCB316 only has a 1.81:1 contrast ratio with the page background (white).

As it is used for the primary focus state outline colour (with no other differentiator) I believe this fails to be an an adequate focus indicator (Focus appearance) as it's below 3:1 and has been overridden from the user agent style.

In some instances this colour is also used for states of interactive components like the button in tabs. Even though it's only on the hover state, I believe Non-text contrast is still appropriate here. Although the states themselves don't need to be differentiated I believe any state of an active user interface component still needs to meet 3:1.

This colour is also used as a "selected" state on the navigation and a hover state on links. This usage appears to be fine as it it doesn't matter if it's there or not as the underline is removed.

**Comment by LJWatson** [comment]
Hello @dashouse  and thanks for filing this issue.

This is no longer the home for the IDP (it's complicated). You'll now find them here:
https://inclusivedesignprinciples.info

With the repo here:
https://github.com/inclusive-design-principles/inclusive-design-principles

If the same issue exists (which is probably does), could I trouble you to open an issue over there?

With thanks.

**Comment by dashouse** [comment]
thanks @LJWatson - have raised it there now
